crates/backend/src/main.rs

```diff
@@ -24,11 +25,17 @@ use actix_web::{
 async fn main() -> std::io::Result<()> {
     let serve_from = "../../packages/playground/dist";
```

Comment:
Same as below, but it will be handled by https://github.com/paritytech/ink-playground/pull/31